From 3721677b5fb7fa1fad7353df7f498f09c8bbb50f Mon Sep 17 00:00:00 2001
From: Nina Gertskin <ngertskin@habana.ai>
Date: Mon, 13 Jan 2025 12:35:37 +0200
Subject: [PATCH] [SW-0] ci: local_run: remove downloading sources

we download sources using manifest

Change-Id: I007c905d26b377c3c1f1a5c8a52a964e378b47a0
Signed-off-by: Nina Gertskin <ngertskin@habana.ai>
(cherry picked from commit 8ca2228db977aa1a2ba1edc53e2ba7e8de689988)
---

diff --git a/scripts/tests/local_run b/scripts/tests/local_run
index 1f3ce22..66f1006 100755
--- a/scripts/tests/local_run
+++ b/scripts/tests/local_run
@@ -40,52 +40,8 @@
 	fi
 }
 
-function local_dl_and_co_repo() {
-	local repo=$1
-	local remote=$2
-	local res=0
-
-	echo running git clone -v ${remote}/${repo} ...
-	git clone -v ${remote}/${repo}
-	pushd ${repo}
-
-	if [ -n "${version}" ]; then
-		git ls-remote --exit-code --heads origin ${version}
-		if [ $? -eq 0 ]; then
-			git remote update
-			git checkout -f origin/${version}
-			res=$?
-		fi
-	fi
-	popd
-
-	return ${res}
-}
-
 function local_prepare_env() {
-	local res1=0
-	local res2=0
-	local res3=0
-
 	print_msg "Start Preparing Environment"
-
-	mkdir -p $stack			|| return 1;
-
-	print_msg "Start Downloading Sources"
-	pushd $stack
-	local_dl_and_co_repo automation ssh://gerrit:29418
-	res1=$?
-	local_dl_and_co_repo habanalabs ssh://git@github.com/HabanaFS
-	res2=$?
-	local_dl_and_co_repo nic_kmd ssh://git@github.com/HabanaFS
-	res3=$?
-	popd
-	if [[ ${res1} -ne 0 || ${res2} -ne 0 || ${res3} -ne 0 ]]; then
-		print_msg "Failed Downloading Sources"
-		return 1
-	fi
-
-	print_msg "Finished Downloading Sources"
 
 	cp $stack/automation/habana_scripts/.bashrc $HOME/		|| return 1;
 	cp $stack/automation/habana_scripts/.bash_aliases $HOME/	|| return 1;
